lwgenerate/gcp/gcp.go

```diff
@@ -404,6 +454,27 @@ func createLaceworkProvider(args *GenerateGcpTfConfigurationArgs) (*hclwrite.Blo
 	return nil, nil
 }
 
+func sortedStringSet(list []string) []string {
+	// TODO: Find me a home
```

Comment:
It would be good to get @dmurray-lacework's input on this. Maybe we could add a helper file to the lwgenerate root directory

Reply:
Mmm yeh, the go-sdk makes use of the internal package for shared 'generic' functions. I would say for now create lwgenerate/internal/array. Nit as well when you do that, can you use Go terminology for the name; slice rather than set.

https://dave.cheney.net/2019/10/06/use-internal-packages-to-reduce-your-public-api-surface

Reply:
Could also be a better idea for lw_generate to have it's own internal package.

Reply:
Placed in `internal/set/strings.go`